tests/e2e/api/transactionResponse.test.ts

```diff
@@ -0,0 +1,530 @@
+          },
+          type: "write_resource",
+        },
+        {
+          address: "0x1",
+          state_key_hash: "0x8048c954221814b04533a9f0a9946c3a8d472ac62df5accb9f47c097e256e8b6",
+          data: {
+            type: "0x1::stake::ValidatorPerformance",
+            data: {
+              validators: [
+                {
+                  failed_proposals: "0",
+                  successful_proposals: "1",
+                },
+              ],
+            },
+          },
+          type: "write_resource",
+        },
+        {
+          address: "0x1",
+          state_key_hash: "0x7b1615bf012d3c94223f3f76287ee2f7bdf31d364071128b256aeff0841b626d",
+          data: {
+            type: "0x1::timestamp::CurrentTimeMicroseconds",
+            data: {
+              microseconds: "1708575651139233",
+            },
+          },
+          type: "write_resource",
+        },
+        {
+          address: "0x1",
+          state_key_hash: "0xacb6a6ca3f454dc5ce4feee8644807c3083559c23b442c4edaa47cfc3e2f8929",
+          data: {
+            type: "0x1::state_storage::StateStorageUsage",
+            data: {
+              epoch: "2",
+              usage: {
+                bytes: "698048",
+                items: "161",
+              },
+            },
+          },
+          type: "write_resource",
+        },
+      ],
+      id: "0x98236644cc6d0169d58492db6eac038fe21bdefe6909e06494753710615e173a",
+      epoch: "2",
+      round: "1",
+      events: [
+        {
+          guid: {
+            creation_number: "3",
+            account_address: "0x1",
+          },
+          sequence_number: "2",
+          type: "0x1::block::NewBlockEvent",
+          data: {
+            epoch: "2",
+            failed_proposer_indices: [],
+            hash: "0x98236644cc6d0169d58492db6eac038fe21bdefe6909e06494753710615e173a",
+            height: "2",
+            previous_block_votes_bitvec: "0x00",
+            proposer: "0xfc1b1b4ed23ac63a5a80121590343578ce704901028322e827b254cc8e404558",
+            round: "1",
+            time_microseconds: "1708575651139233",
+          },
+        },
+      ],
+      previous_block_votes_bitvec: [0],
+      proposer: "0xfc1b1b4ed23ac63a5a80121590343578ce704901028322e827b254cc8e404558",
+      failed_proposer_indices: [],
+      timestamp: "1708575651139233",
+      type: TransactionResponseType.BlockMetadata,
+    } satisfies BlockMetadataTransactionResponse;
+  });
+
+  //   Comment out this test until the API is fixed for single sender txns
+  //   test("it tests a sample user_transaction JSON response satisfies UserTransactionResponse", async () => {
+  //    const txn = {
+  //       "version":"6224",
+  //       "hash":"0xa17bffc240c11f8e9bc2ba8b4b1a45cae44846303c96bf62ca9fafa0fc2d07a8",
+  //       "state_change_hash":"0x2d5c8a6f05e820e81b297b3cde83bf55f3e5ea86f6edc2dc11744e851beea5cb",
+  //       "event_root_hash":"0x1b7628f4bbd261df56e25b44bdce7665193e5014c8221481f631927e1c7974ef",
```

Comment:
? Why add a commented out test?

What does this mean "API is fixed for single sender txns"?

Reply:
It fails because the API wasnt updated to support single sender transactions when @davidiw  implemented it.  This was the issue that bowen encountered.

Reply:
not following.. why do we add a non-working commented out test? can't we just not add it..?

Reply:
Yeah we either add it, how it looks today, or we delete the commented out test.

Reply:
deleted